gas/load_gas.yml

```diff
@@ -0,0 +1,65 @@
+methods:
+  eth_accounts: 1
+  eth_chainId: 1
+  eth_syncing: 1
+  net_version: 1
+  net_listening: 1
+  net_peerCount: 1
+  eth_protocolVersion: 1
+  eth_hashrate: 1
+  eth_coinbase: 1
+  eth_mining: 1
+  web3_clientVersion: 1
```

Comment:
Why do these methods have 1 instead of 0? In dproxy they have 0 in load_gas file